perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -1243,6 +1243,8 @@ public static void makeUserPerunAdmin(PerunSession sess, User user) throws Alrea
 			throw new RoleCannotBeManagedException(role, complementaryObject);
 		}
 
+
+
```

Comment:
Is there any reason for these empty lines?

Reply:
Fixed